change/@ni-nimble-components-bcea55bf-ed59-4b11-b80b-6311749da460.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Adding filter input to Select",
+  "packageName": "@ni/nimble-components",
+  "email": "26874831+atmgrifter00@users.noreply.github.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
FYI I see a fair bit of open feedback on the PR so I haven't prioritized reviewing it. Wanna make sure we didn't deadlock waiting on each other to review.